doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -250,26 +248,24 @@ Other configurations set in the devicetree, such as the current speed, are overw
 
 Modem fault handling
 ********************
-If a fault occurs in the modem, the application is notified through the fault handler function that is registered with the Modem library during initialization.
-This enables the application to read the fault reason (in some cases the modem's program counter) and take appropriate action.
+When a fault occurs in the modem, the application is notified through the fault handler function that is registered with the Modem library during initialization.
```

Comment:
`When` makes it sound like this is something we expect to happen.
```suggestion
If a fault occurs in the modem, the application is notified through the fault handler function that is registered with the Modem library during initialization.
```

Reply:
Ok, updated